cmd/tether/attach.go

```diff
@@ -584,3 +586,50 @@ type winsize struct {
 	wsXpixel uint16
 	wsYpixel uint16
 }
+
+type stdinReader struct {
+	reader io.ReadCloser
+}
+
+func newStdinReader(channel ssh.Channel) *stdinReader {
+	r, w := io.Pipe()
+	s := &stdinReader{
+		reader: r,
+	}
+
+	go func() {
+		buf := make([]byte, 512)
```

Comment:
I would increase a buffer size to 2k, when we migrate from serial port based solution larger buffer is better for performance.